### DISCUSSION TITLE
Discord server for more informal chat :)

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#629] [General]
Hey folks, just letting you know that I've created a discord server for twinBASIC!  I really have no idea what I'm doing, but I figured we could give it a go...

https://discord.gg/UaW9GgKKuE

**_(NOTE: if the link says expired, please first register an account at Discord and login, and you should then find the link works)_**

Being a brand-new server, it's lonely over there right now, so why not pop in and say hello!   There's no plan to change things over here... this GitHub repo will still be the best place to put your bug reports and feature requests.   

Things are less formal over there, so if you want any general help with twinBASIC, or have some outrageous ideas that you want to bounce off a few people, you're very welcome to come over to the discord server to discuss.

**Comment by mburns08109** [comment]
FYI - certain commercial web filters block Discord altogether, so enterprise-level folks may not be able to access your discord server.
...just something to keep in mind before building a major dependency on it.

**Reply by WaynePhillipsEA** [reply]
Thanks for the info!  I expect most of the discussions will continue on here anyway.  The Discord server will likely be used to thrash out ideas before they make it to this repo.

**Reply by skytech2024** [reply]
i just encounter an error using the twinBasic IDE BETA 982. i was changing font type on a report header, and report runs into error and report design wiped out

**Reply by WaynePhillipsEA** [reply]
This is not the right place to report an issue.  Please open up a bug report and include details such as steps to reproduce the problem:   https://github.com/twinbasic/twinbasic/issues/new/choose

**Comment by skytech2024** [comment]
Noted, I have done that

*Sala E.Y.*
***@***.***
